spec/services/update_package_price_spec.rb

```diff
@@ -33,12 +33,26 @@
   # This tests covers feature request 1. Feel free to add more tests or change
   # the existing one.
 
-  xit "supports adding a price for a specific municipality" do
+  it "supports adding a price for a specific municipality if municipality not exists" do
```

Comment:
You should use `context` to avoid conditions in spec names

Reply:
I agree it's good to use `context`. I just tried it to match the already used convention.